Key.h

```diff
@@ -26,12 +26,23 @@ THE SOFTWARE.
 #ifndef __OPENPGP_KEY__
 #define __OPENPGP_KEY__
 
+#include <set>
 #include "Packets/packets.h"
 #include "PKA/PKAs.h"
 #include "PGP.h"
 
 namespace OpenPGP {
     class Key : public PGP {
+        public:
+            typedef std::multimap<Packet::Tag::Ptr, Packet::Tag::Ptr> sigPairs;
+
+            struct pkey{
```

Comment:
What does the `p` stand for?

Reply:
I have called that struct pkey because is the same nomenclature used in SKS (written in ocaml). The idea of the merge function has been taken from there.

I will solve the other problem and re-commit the file